modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas-describe.adoc

```diff
@@ -0,0 +1,67 @@
+= rpk cluster quotas describe
+:description: rpk cluster quotas describe
+
+Describe client quotas.
+
+This command describes client quotas that match the provided filtering criteria. Running the command without filters returns all client quotas. Use the
+`--strict` flag for strict matching, which means that the only quotas returned  exactly match the filters.
+
+Filters can be provided in terms of entities. An entity consists of either a client ID or a client ID prefix.
+
+== Usage
+
+[,bash]
+----
+rpk cluster quotas describe [flags]
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|--any |strings |type for any matching (names or default), where type is client-id or client-id-prefix (repeatable).
+
+|--default |strings |type for default matching, where type is client-id or client-id-prefix (repeatable).
+
+|--format |string |Output format. Possible values: `json`, `yaml`, `text`, `wide`, `help`. Default: `text`.
+
+|-h, --help |- |Help for describe.
+
+|--name |strings |type=name pair for exact name matching, where type is client-id or client-id-prefix (repeatable).
+
+|--strict |- |whether matches are strict, if true, entities with unspecified entity types are excluded.
+
+|--config |string |Redpanda or `rpk` config file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

Comment:
```suggestion
|--config |string |Redpanda or `rpk` config file. Default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

Reply:
Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.